api/services/elastigroup/azure/scaleSetVms/schemas/lowPriorityVMsEGCreation.yaml

```diff
@@ -0,0 +1,1202 @@
+                  <li>Microsoft.Network/virtualNetworkGateways</li>
+                  <li>Microsoft.Network/expressRouteCircuits</li>
+                  <li>Microsoft.Network/expressRouteCircuits/peerings</li>
+                  <li>Microsoft.Network/connections</li>
+                  <li>Microsoft.Network/trafficManagerProfiles</li>
+                  <li>Microsoft.Network/networkWatchers/connectionMonitors</li>
+                  <li>Microsoft.Network/frontdoors</li>
+                  <li>Microsoft.NotificationHubs/Namespaces/NotificationHubs</li>
+                  <li>Microsoft.OperationalInsights/workspaces</li>
+                  <li>Microsoft.PowerBIDedicated/capacities</li>
+                  <li>Microsoft.Relay/namespaces</li>
+                  <li>Microsoft.Search/searchServices</li>
+                  <li>Microsoft.ServiceBus/namespaces</li>
+                  <li>Microsoft.SignalRService/SignalR</li>
+                  <li>Microsoft.Sql/servers/databases</li>
+                  <li>Microsoft.Sql/servers/elasticPools</li>
+                  <li>Microsoft.Sql/managedInstances</li>
+                  <li>Microsoft.Storage/storageAccounts</li>
+                  <li>Microsoft.Storage/storageAccounts/blobServices</li>
+                  <li>Microsoft.Storage/storageAccounts/fileServices</li>
+                  <li>Microsoft.Storage/storageAccounts/queueServices</li>
+                  <li>Microsoft.Storage/storageAccounts/tableServices</li>
+                  <li>Microsoft.StreamAnalytics/streamingjobs</li>
+                  <li>Microsoft.TimeSeriesInsights/environments</li>
+                  <li>Microsoft.TimeSeriesInsights/environments/eventsources</li>
+                  <li>Microsoft.Web/serverfarms</li>
+                  <li>Microsoft.Web/sites (excluding functions)</li>
+                  <li>Microsoft.Web/sites (functions)</li>
+                  <li>Microsoft.Web/sites/slots</li>
+                  <li>Microsoft.Web/hostingEnvironments/multiRolePools</li>
+                  <li>Microsoft.Web/hostingEnvironments/workerPools</li>
+                  </ul>"
+                type: string
+                example: "Microsoft.Network/applicationGateways"
+                default: Microsoft.Compute
+              operator:
+                title: Operator
+                description: "The operator used to evaluate the threshold against the current metric value.
+                   <ul>
+                   <li>gt (Greater Than)</li>
+                   <li>gte (Greater Than or Equal)</li>
+                   <li>lt (Less Than)</li>
+                   <li>lte (Less Than or Equal)</li>
+                   <ul>"
+                type: string
+                example: "gte"
+                enum:
+                  - gt
+                  - gte
+                  - lt
+                  - lte
+              dimenstions:
+                title: Dimenstions
+                description: TBD
```

Comment:
@vlad666spot please fix TBD across this schema

Reply:
@sagishoshan fixed